src/web/teap/requirements/prod.txt

```diff
@@ -0,0 +1,38 @@
+# Everything needed in production
```

Comment:
Ditto here, some dependencies are second or third order. Let's have a `prod.in` with the essential requirements if it is possible.

Reply:
Yes, will do